packages/core/src/browser/widgets/widget.ts

```diff
@@ -302,9 +307,7 @@ export function addKeyListener<K extends keyof HTMLElementEventMap>(
     }));
     for (const type of additionalEventTypes) {
```

Comment:
thought: Should we filter 'keydown' from the additionalEventTypes so adopters may not register the same handler twice like
```
        this.addKeyListener(this.node, Key.ESCAPE, event => {
            console.log(event.ctrlKey);
        }, 'keydown');
```

But that would also change the previous behavior so probably we should just keep the current behavior.